src/Components/Form.js

```diff
@@ -29,36 +29,40 @@ const Form = () => {
   };
 
   return (
-    <section>
-      <h2>Add Book</h2>
-      <form>
-        <input
-          type="text"
-          value={setnewBook.title}
-          onChange={onChange}
-          name="title"
-          placeholder="Choose Book's Title.."
-          required
-        />
-        <input
-          type="text"
-          value={setnewBook.author}
-          onChange={onChange}
-          name="author"
-          placeholder="Choose Book's Author.."
-          required
-        />
-        <button
-          type="button"
-          onClick={async () => {
-            await dispatch(addBook(newBook));
-            setCount(add);
-          }}
-        >
-          Add
-        </button>
-      </form>
-    </section>
+    <>
+      <div className="divider" />
+      <section>
+        <h2>Add New Book</h2>
+        <form>
+          <input
+            type="text"
+            value={setnewBook.title}
+            onChange={onChange}
+            name="title"
+            placeholder="Choose Book Title..."
+            required
+          />
+          <input
+            type="text"
+            value={setnewBook.author}
+            onChange={onChange}
+            name="author"
+            placeholder="Choose Book Author..."
+            required
+          />
+          <button
+            className="form-button"
+            type="button"
+            onClick={async () => {
+              await dispatch(addBook(newBook));
+              setCount(add);
+            }}
+          >
+            Add
+          </button>
+        </form>
```

Comment:
- Good job done so far, but this button is not styled like it is in the design template, please fix this 🙏. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏
Your project:
![tjsths](https://github.com/vangelif/bookstore/assets/73607512/35638d17-9ab8-4c39-88e0-2351cbf0c287)

Design template:
![ehehs](https://github.com/vangelif/bookstore/assets/73607512/85974f84-93b2-4388-941f-58260863691b)